app/Http/Controllers/AccountController.php

```diff
@@ -74,6 +79,12 @@ public function update(Request $request)
         $user->allow_profile_contact = $request->get('allow_profile_contact');
         $user->profile_intro = $request->get('profile_intro');
         $user->profile_slug = $request->get('profile_slug');
+
+        if ($request->file('profile_picture')) {
+            $image = $request->file('profile_picture');
+            Event::fire(new ProfilePictureUpdated($user, $image->getRealPath(), $image->getClientOriginalExtension()));
```

Comment:
I know that methods are supposed to have a single responsibility, that's why I've put it in an event. 

Should I keep the manipulation in the event but move the file to storage in the method, remove the event and keep it all in the method, or remove the ShouldQueue?